src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -688,6 +689,17 @@ public LDAPConfiguration getConfigurationFor(String configurationId) {
         return null;
     }
 
+    @CheckForNull
+    private static LDAPConfiguration _getConfigurationFor(String configurationId) {
+        final Jenkins jenkins = Jenkins.getInstance();
+        final SecurityRealm securityRealm = jenkins == null ? null : jenkins.getSecurityRealm();
```

Comment:
Looks like FindBugs is failing the build against 2.73.1 because of the `Jenkins#getInstance` changes:

    Redundant nullcheck of jenkins, which is known to be non-null in hudson.security.LDAPSecurityRealm._getConfigurationFor(String) [hudson.security.LDAPSecurityRealm] Redundant null check at LDAPSecurityRealm.java:[line 695] RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE

Is there a way to disable FindBugs for the 2.73.1 build? Against 1.651.3 (baseline in pom.xml) the null check is necessary.